index/get_manifest_info.py

```diff
@@ -50,20 +53,21 @@ def _harvest_important_manifest_info(manifest_json, manifest_info):
 
 
 def _get_manifest_repository(manifest_json):
-    repository = ""
-    logo = _get_manifest_logo(manifest_json)
-    if 'Archive'.upper() in logo.upper():
+    """ Find Repository from Manifest Attribution """
+    attribution = ''
+    repository = 'SNITE'  # set default
+    attribution = _get_manifest_attribution(manifest_json)
+    if 'Archives'.upper() in attribution.upper():
```

Comment:
Could be rewritten to use a dictionary(see other review comment below for example) instead or use if/elif/else instead of just if to save some cycles.